Misc/NEWS

```diff
@@ -338,6 +338,8 @@ Extension Modules
 
 Library
 -------
+- bpo-30274: Rename 'name' to 'fullname' argument to ExtensionFileLoader. Patch
```

Comment:
We don't edit `Misc/NEWS` manually anymore. Please use blurb to create a NEWS entry: https://github.com/python/core-workflow/tree/master/blurb